.travis.yml

```diff
@@ -1,4 +1,10 @@
 language: ruby
-cache: bundler
+script:
+  - bundle exec rake rubocop
 rvm:
   - 2.3.3
+sudo: false
+cache: bundler
+script:
```

Comment:
I've removed the `rubocop` script entry in a fix up. Rubocop is part of the default Rake script so I assume I don't need to call it explicitly.